iron-ajax.html

```diff
@@ -235,6 +235,18 @@
         readOnly: true
       },
 
+      /**
+       * The `progress` property of this element's `lastRequest`.
+       */
+      lastProgress: {
+        type: Object,
+        notify: true,
+        readOnly: true,
+        value: function() {
+          return {};
```

Comment:
question: why default this to an empty object?

Reply:
I agree

Reply:
In that case, mind updating it both here (as in remove the value: ... part) and when it gets reset before `request.send(requestOptions);`?